Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+pipeline {
+    agent { label 'pr' }
+
+    options {
+        timestamps()
+        timeout(time: 1, unit: 'HOURS')
+        disableConcurrentBuilds(abortPrevious: true)
+    }
+
+    stages {
+        stage('Validate commit') {
+            steps {
+                script {
+                    def CHANGE_REPO = sh(script: 'basename -s .git `git config --get remote.origin.url`', returnStdout: true).trim()
+                    build job: '/Utils/Validate-Git-Commit', parameters: [
+                        string(name: 'Repo', value: "${CHANGE_REPO}"),
+                        string(name: 'Branch', value: "${env.CHANGE_BRANCH}"),
+                        string(name: 'Commit', value: "${GIT_COMMIT}")
+                    ]
+                }
+            }
+        }
+
+        stage('Build') {
+            steps {
+                sh 'make'
+            }
+        }
+
+        stage('Run tests') {
+            steps {
+                sh 'go test ./...'
+            }
+        }
+
+        stage('Clean up') {
```

Comment:
There is a little issue here: failing builds will not clean up environment. Leaving a directory behind (never to be reused).  
I will address this some other time, maybe with
```
    post {
                always {
   // do cleanup
                }
      }
 ```

Reply:
Thanks, fixed